test/provisioner/provision_ibmcloud.properties

```diff
@@ -0,0 +1,32 @@
+# Manage -> Access -> API Keys -> My IBM Cloud API Keys
+APIKEY="${MY_VPC_APIKEY}"
+CLUSTER_NAME="e2e-test1"
+# Resource list -> storage -> a cos service ->
+COS_BUCKET="peerpod-cos-bucket"
+# Resource list -> storage -> a cos service -> instances -> an cos service instance -> service credentials -> resource_instance_id
+COS_INSTANCE_ID="crn:v1:bluemix:public:cloud-object-storage:global:a/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx::"
+# Resource list -> cloud object storage -> endpoints
+COS_SERVICE_URL="s3.jp-tok.cloud-object-storage.appdomain.cloud"
+# Resource list -> storage -> a cos service -> instances -> an cos service instance -> service credentials -> apikey
+COS_APIKEY="${MY_COS_SERVICE_APIKEY}"
+IAM_SERVICE_URL="https://iam.cloud.ibm.com/identity/token"
+IKS_VERSION="1.26"
+IS_SELF_MANAGED_CLUSTER="no"
+INSTANCE_PROFILE_NAME="bz2-2x8"
```

Comment:
I guess this isn't used until we support self-managed cluster, so maybe comment it out for now?